WickedEngine/wiInput.cpp

```diff
@@ -548,6 +548,115 @@ namespace wiInput
                 case wiInput::KEYBOARD_BUTTON_PAGEUP:
                     keycode = SDL_SCANCODE_PAGEUP;
                     break;
+				case 48:
```

Comment:
just one small thing, could you include in a comment a link where this codes come from? for future refence

Reply:
Done!